app/src/App.js

```diff
@@ -1,11 +1,21 @@
 import React, { Component } from 'react';
 import './App.scss';
+import { ClassNames } from './component-library/helpers/classNames';
+
+
+
 
 class App extends Component {
+  state = {
+    blk: 'app',
+    elt: 'sub',
+    mods: ['class1', 'class2']
+  };
```

Comment:
why is this needed? 
Why can't we send the object directly to the function?

Reply:
I created a component that's why I used state.  I will convert it in utility function and pass object in it.

Reply:
className({ blk: 'app', elt: 'sub', mods: [ 'class1', 'class2' ]})